.github/workflows/cancel.yml

```diff
@@ -0,0 +1,14 @@
+name: Cancel obsolete workflows
+
+on: [push]
+
+jobs:
+  cancel-obsolete-workflows:
+    name: 'Cancel obsolete workflows'
+    runs-on: macos-latest
+    timeout-minutes: 3
+    steps:
+      - uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          workflow_id: beta, ci # Workflow file names to cancel
+          access_token: ${{ github.token }}
```

Comment:
Can we somehow bump the priority of this action? If there are already 20 actions in the queue, it won't help that this will be executed as the 21st :) Or am I missing somethign?

Reply:
Probably makes sense? cc @nuno-vieira

Reply:
Yes I'll add this to the other jobs 👍